telegram/_passport/encryptedpassportelement.py

```diff
@@ -40,7 +40,7 @@ class EncryptedPassportElement(TelegramObject):
     by the user. The data has been automatically decrypted by python-telegram-bot.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
-    considered equal, if their :attr:`type`, :attr:`data`, :attr:`phone_number`, :attr:`email`,
+    considered equal, if their :attr:`type`, :attr:`data`, :paramref:`phone_number`, :attr:`email`,
```

Comment:
um shouldn't the others also be paramrefs? Same in all files

Reply:
No, equality is determined by the attributes which (in general) may have little to do with the arguments. This change is probably due to me search-replacing a bit too thoroughly … Reverted